server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -272,6 +302,14 @@ export class TenantManager {
             throw new NetworkError(500, "Tenant key2 decryption failed.");
         }
 
+        const cacheKeys = JSON.stringify({
+            key1: encryptedTenantKey1,
+            key2: encryptedTenantKey2,
+        });
+
+        await this.cache.set(tenantId, cacheKeys);
```

Comment:
We should use a slightly more detailed cache key, rather than simply "tenantId". Since the Redis instance is possibly shared with other services, we should not rely on other services to not use "tenantId" for a cache key. Instead, adding a prefix like `tenantKeys:${tenantId}` to the cache key would be preferable